src/main/java/com/volasoftware/tinder/constants/Constants.java

```diff
@@ -0,0 +1,6 @@
+package com.volasoftware.tinder.constants;
+
+public class Constants {
```

Comment:
Name of classes in Java are usually singular.

Reply:
File was removed.